big-structure/src/destress_big_structure/analysis.py

```diff
@@ -256,7 +256,17 @@ def analyse_design(design: ampal.Assembly) -> DesignMetrics:
     assert (
         AGGRESCAN3D_SCRIPT_PATH
     ), "AGGRESCAN3D_SCRIPT_PATH is not defined, check you `.env` file"
-    ev.tag_dssp_data(design)
+
+    try:
+        ev.tag_dssp_data(design)
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(
+            "command '{}' return with error (code {}): {}".format(
+                e.cmd, e.returncode, e.output
+            )
+        )
+        print("Error with Isambard Tag DSSP")
```

Comment:
If you raise a `RuntimeError`, the print statement will be unreachable, the whole process will panic and exit. Wouldn't it be better to suppress the error and return the error string to the user?